guide/case-studies/wallet-scrutiny.md

```diff
@@ -0,0 +1,215 @@
+   layout = "full-width"
+%}
+
+#### 6. Redesign
+A new homepage design was then created and shared with the team. The new design had the following elements.
+
+##### Branding:
+The brand direction was designed to feel functional and inclusive yet trustworthy. The logo of the dog with its ears up came about from an idea that the WalletScrutiny team/website would bark when they spotted a wallet that they tested which did not pass all the tests.
+
+**Why**:
+The feeling that a user would get when arriving at the homepage would largely be determined by the tagline as well as the overall feel and branding. We wanted to use color and copy to create an inviting and trustworthy feel for users.
+
+##### Homepage:
+The copy and UI design was something that we focused on a lot for the homepage. Copy, being an essential part of design, was tricky with a project like Wallet Scrutiny. We had the help of a copy ninja who helped us to create a catchy headline as well as revise all the copy on the homepage and methodology page. The homepage was designed with a strong call to action with the search bar as well as a nice tile overview of the wallets on which reviews have been done. Showing the wallets that passed all the tests right from the homepage and allowing  a filter option to sort through the reviews.
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/case-studies/wallet-scrutiny/homepage-before-after-wallet-scrutiny.png"
+   retina = "/assets/images/guide/case-studies/wallet-scrutiny/homepage-before-after-wallet-scrutiny@2x.png"
+   alt-text = "Two side by side images. The first image is the homepage before the re-design, the second page is the homepage after the re-design."
+   width = 400
+   height = 89
+   layout = "float-left-desktop"
+%}
+
+**Why**:The strong call to action of typing into the green search box was intentional to allow the user to search straight away for the wallet they are using or were thinking of using.
+
+Essentially there are four main categories that wallets fall into; Android, iPhone, hardware and bearer. We wanted to create a quick filter option of this as well as present a default view of the wallets that passed all the tests.
+
+</div>
+
+##### Methodology page:
+The methodology page was revised with a focus on two directions. We revised the copy as well as the navigation structure. The page was broken up into three sections. Introduction, tests we run as well as FAQ’s.
+
+**Why**:
+The original methodology page was a single page outlining the full process.  Trying to find a simple, easy-to-read structure for this diverse mix of security data and test results was a unique challenge. Since the process to evaluate wallets is a lengthy one we decided to break this up into three distinct sections. This change allowed for easier readability and navigation. The copy of the methodology page was also worked on to improve readability by looking for opportunities to simplify the content.
```

Comment:
```suggestion
The original methodology page was a single page outlining the full process. Trying to find a simple, easy-to-read structure for this diverse mix of security data and test results was a unique challenge. Since the process to evaluate wallets is a lengthy one, we decided to break it up into three distinct sections. This change allowed for better readability and navigation. The copy of the methodology page was also worked on to improve readability, by looking for opportunities to simplify the content.
```